### PR TITLE
Coq and Coqide packages for 8.13+beta1

### PR DESCRIPTION
**Pull request by gares**
CC @erikmd

### CODE REVIEW

**Comment by erikmd** [comment]
Thanks @gares.

Maybe one should wait that the opam-repository PR is merged https://(github.com/ocaml/opam-repository/pull/17690#issuecomment-739922936) for merging this one.

And at that time, restart the CI of this PR #1496 ?

**Comment by gares** [comment]
does `depopt` pull in by default?

**Comment by erikmd** [comment]
I'm unsure actually… well, we'll see this thanks to opam-coq-archive CI :)

**Comment by gares** [comment]
w.r.t CI here, I think we should add 1 job (pick one compiler) and in that job force coq-native to be installed

**Comment by gares** [comment]
in the others, have it off (maybe some option options are needed, maybe not, as you I don't know the default)

**Comment by gares** [comment]
Are these package OK, in your opinion? (I'm start building the windows installer out of these, in the meanwhile)

**Comment by erikmd** [comment]
> w.r.t CI here, I think we should add 1 job (pick one compiler) and in that job force coq-native to be installed

Indeed, why not @gares, let's say the 4.07* flavor of the compiler?

Regarding how to set the native flag off, it suffices to do nothing (not doing `opam install coq-native`)

**Comment by erikmd** [comment]
> Are these package OK, in your opinion? (I'm start building the windows installer out of these, in the meanwhile)

Sorry I didn't have a look yet (I've some urgent task to complete up to 16:00 normally) so I guess I'l let the opam-coq-archive maintainers to proofread in the meantime !-)

**Comment by gares** [comment]
ok!

**Comment by gares** [comment]
I was merged

**Comment by erikmd** [comment]
@palmskog @gares indeed: the coq-native package has been deployed, so we get:

```
$ opam show coq-native

<><> coq-native: information on all versions ><><><><><><><><><><><><><><><><><>
name         coq-native
all-versions 1
[…]
```

see also https://opam.ocaml.org/packages/coq-native

@palmskog do you think the CI should be restarted in this PR just to perform another test?

and when do you think that the extension of the CI suggested below should be done? probably in another PR…

https://github.com/coq/opam-coq-archive/pull/1496#issuecomment-739927816
> > w.r.t CI here, I think we should add 1 job (pick one compiler) and in that job force coq-native to be installed
> 
> Indeed, why not @gares, let's say the 4.07* flavor of the compiler?
> 
> Regarding how to set the native flag off, it suffices to do nothing (not doing `opam install coq-native`)

**Comment by gares** [comment]
(CI improvement in another PR please)

**Comment by gares** [comment]
I'll restart one job, to see how it goes

**Comment by silene** [comment]
> does `depopt` pull in by default?

No, it does not. (Sorry if this was already mentioned and I missed it.)

**Comment by erikmd** [comment]
OK! anyway, CI is green :heavy_check_mark:

File: core-dev/packages/coq/coq.8.13+beta1/opam
Comment (erikmd) [inline core-dev/packages/coq/coq.8.13+beta1/opam:24]: Should this be
```suggestion
  "ocaml" {>= "4.05.0" & < "4.12"}
```
(like `opam show -f depends: coq.8.12.1` ?)

File: core-dev/packages/coqide/coqide.8.13+beta1/opam
Comment (erikmd) [inline core-dev/packages/coqide/coqide.8.13+beta1/opam:31]: @gares @proux01 @Zimmi48 @palmskog, sorry to ping you… do you think the `-native-compiler` option is needed for the coqide package?

Comment (gares) [reply core-dev/packages/coqide/coqide.8.13+beta1/opam:31]: I did set it to keep the invariant that the configure is called with the same options. I believe it is not important.

Comment (gares) [reply core-dev/packages/coqide/coqide.8.13+beta1/opam:31]: My rationale here is that:
- if it is irrelevant, it won't hurt
- if CoqIDE starts to understand something about native, then it is better to be configured this way

**Review by erikmd — APPROVED** [review]

**Comment by gares** [comment]
We are waiting for the windows installer to be signed by Inria, it should happen tomorrow.
After that, I'd like to announce the beta (and merge this one).

**Comment by erikmd** [comment]
OK great :+1: (and regarding the docker-coq packaging, as coq-bignums.8.13+beta1 is already available, I'll be able to update the 8.13 docker image as soon as this PR is merged)

BTW don't forget to reply my last question in this issue − https://github.com/coq/bignums/issues/52#issuecomment-739524437 (albeit it is not urgent :)

**Pull request merged**
